libgeotiff/bin/listgeo.c

```diff
@@ -227,20 +226,29 @@ static void GTIFPrintCorners( GTIF *gtif, GTIFDefn *defn, FILE * fp_out,
 
 {
     printf( "\nCorner Coordinates:\n" );
+
+    unsigned short raster_type = RasterPixelIsArea;
+    GTIFKeyGet(gtif, GTRasterTypeGeoKey, &raster_type, 0, 1);
+
+    double xmin = (raster_type == RasterPixelIsArea) ? 0.0 : -0.5;
+    double ymin = xmin;
+    double ymax = ymin + ysize;
```

Comment:
should likely be ysize + 1 in the Point case. And similarly for xsize + 1

Reply:
I think the version is correct as it is. There are ysize pixels of the same size independent of the Point/Area distinction, and you just have to offset the border for the Point case. See the ASCII-drawing here: http://geotiff.maptools.org/spec/geotiff2.5.html#2.5.2

Reply:
you're probably right. Did you compare with the output of gdalinfo ?

Reply:
Yes, it appears to be identical. For the example used in the report, where top left is (-0.04,-81.38)  and bottom right (-18.45,-68.65):

listgeo without fix:
```
Upper Left    (-81.3800000,-0.0400000)
Lower Left    (-81.3800000,-18.4807346)
Upper Right   (-68.6287479,-0.0400000)
Lower Right   (-68.6287479,-18.4807346)
Center        (-75.0043740,-9.2603673)
```

listgeo with fix:
```
Upper Left    (-81.3906260,-0.0246327)
Lower Left    (-81.3906260,-18.4653673)
Upper Right   (-68.6393740,-0.0246327)
Lower Right   (-68.6393740,-18.4653673)
Center        (-75.0150000,-9.2450000)
```

gdalinfo:
```
Upper Left  ( -81.3906260,  -0.0246327) ( 81d23'26.25"W,  0d 1'28.68"S)
Lower Left  ( -81.3906260, -18.4653673) ( 81d23'26.25"W, 18d27'55.32"S)
Upper Right ( -68.6393740,  -0.0246327) ( 68d38'21.75"W,  0d 1'28.68"S)
Lower Right ( -68.6393740, -18.4653673) ( 68d38'21.75"W, 18d27'55.32"S)
Center      ( -75.0150000,  -9.2450000) ( 75d 0'54.00"W,  9d14'42.00"S)
```

Reply:
great